### PR TITLE
615 check extension

### LINKED ISSUE #615: When user doesn't have extension and clicks on [log-in] button
Right now when a user lands at KodaDot w/o extension isn't aware that needs some extension.
Be smart and came up with a way how to show user dialogue to install the polkadot.js extension  

Inspiration https://polkadot.js.org/extension/
Link to chrome https://chrome.google.com/webstore/detail/polkadot%7Bjs%7D-extension/mopnmbcafieddcagagdcbnhejhlodfdd
Link to firefox https://addons.mozilla.org/en-US/firefox/addon/polkadot-js-extension/

image

### PR DESCRIPTION
**Pull request by gdev1001**
**Thank you for your contribution** to the KodaDot NFT gallery.
Let's do a quick check before the merge.
Your contribution will be an eternal part of the community codebase

### Before submitting this PR, please make sure:
- [ ] Code builds clean without any errors or warnings
- [ ] I've merged recent default branch -- **main** and I've no conflicts
- [ ] I didn't break any original functionality

### Optional
- [ ] I've tested it on mobile and everything works

### PR type
- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new? (maybe part of changelog)
- closes #615 - <Added - check polkadot.js extension on the login page (if you've installed polkadot.js in your browser then when you click the 'Login' button, they should be showing a dropdown to login but if you didn't install polkadot.js then should be showing the dropdown to install polkadot.js )>

### If issue contains bounty label ($,$$,$$$)
- Your KSM address: <Kusama_address>

### Community participation
- [x] Are you at KodaDot Discord? If no, [let's join us - https://discord.gg/35hzy2dXXh](https://discord.gg/35hzy2dXXh)

### Screenshot
- 
Selection_239

### CODE REVIEW

**Comment by yangwao** [comment]
which issues is closed by this? can you update it in PR description please?

File: src/components/rmrk/Profile/NavbarProfileDropdown.vue [hunk 1]
```diff
@@ -65,6 +65,23 @@
...
+      </b-dropdown-item>
+      <b-dropdown-item has-link aria-role="menuitem">
+        <a href="https://chrome.google.com/webstore/detail/polkadot%7Bjs%7D-extension/mopnmbcafieddcagagdcbnhejhlodfdd" target="_blank" class="is-flex is-align-items-center pl-3">
+          Link to chrome
+        </a>
+      </b-dropdown-item>
+      <b-dropdown-item has-link aria-role="menuitem">
+        <a href="https://addons.mozilla.org/en-US/firefox/addon/polkadot-js-extension/" target="_blank" class="is-flex is-align-items-center pl-3">
+          Link to firefox
+        </a>
```
Comment (roiLeo) [inline src/components/rmrk/Profile/NavbarProfileDropdown.vue:82]: Could you add ```rel="noopener noreferrer"``` to ```target="_blank"``` links ?

**Review by yangwao — COMMENTED** [review]
Left some reviews

**Comment by yangwao** [comment]
Awesome!
Screenshot 2021-08-25 at 15 47 03

**Pull request merged**